modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuring-image-registry-settings-hcp_{context}"]
+= Configuring image registry settings for {hcp-title}
+
+Verify that your image registry is configured in the cluster details by running the following command:
```

Comment:
You can configure your image registy settings at cluster creation with the following procedure. The cluster's nodes will be use the desired configuration after creation.

Reply:
added thanks

Reply:
@andreadecorte used this, not sure if correct:
You can configure image registry settings at cluster creation. The cluster's nodes will use the required configuration after creation.